test/e2e/data/shared/v1beta1/metadata.yaml

```diff
@@ -1,6 +1,9 @@
 apiVersion: clusterctl.cluster.x-k8s.io/v1alpha3
 kind: Metadata
 releaseSeries:
+  - major: 1
+    minor: 11
+    contract: v1beta1
```

Comment:
Great, although I think we are missing the `/metadata.yaml` bump: https://github.com/justinsb/cluster-api-provider-gcp/blob/b1d9c054273c19a90f1fe2f71411fe2fdb1fb8dd/metadata.yaml#L45-L46

Reply:
I mentioned it here: https://github.com/kubernetes-sigs/cluster-api-provider-gcp/pull/1509#issuecomment-3236326815

I worry about this ("update this file only when a new major or minor version is released"): https://github.com/justinsb/cluster-api-provider-gcp/blob/b1d9c054273c19a90f1fe2f71411fe2fdb1fb8dd/metadata.yaml#L5

Also, that I would have to say `contract: v1beta1` but I _think_ that we want to try to get to `contract: v1beta2` (or should at least discuss when we want to do that!)

Reply:
It looks like I have to set it for the tests to pass so this might be moot!